borgbackup: 1.1.9 -> 1.1.10

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/borgbackup/versions.



meta.description for borgbackup is: '"A deduplicating backup program (attic fork)"'.


meta.homepage for borgbackup is: '"https://www.borgbackup.org"


<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS
- /bin/sh passed the binary check.
- 1 of 1 passed binary check by having a zero exit code.
- 0 of 1 passed binary check by having the new version present in output.
- found 1.1.10 with grep in /nix/store/4m7p421h0831sq45rlp7xk4fp84vw4xx-borgbackup-1.1.10
- directory tree listing: https://gist.github.com/974d3b69e539e398c88e79618749aa89
- du listing: https://gist.github.com/baae304f0bdfce480f249fbad5777295

</details>
<details>
<summary>
Rebuild report (if merged into master) (click to expand)
</summary>

4 total rebuild path(s)

1 package rebuild(s)

1 x86_64-linux rebuild(s)
1 i686-linux rebuild(s)
1 x86_64-darwin rebuild(s)
1 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
borgbackup

</details>

<details>
<summary>
Instructions to test this update (click to expand)
</summary>

Either download from Cachix:
```
nix-store -r /nix/store/4m7p421h0831sq45rlp7xk4fp84vw4xx-borgbackup-1.1.10 \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)

Or, build yourself:
```
nix-build -A borgbackup https://github.com/r-ryantm/nixpkgs/archive/f4b8412198b33365d39f476b6516453845800557.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/4m7p421h0831sq45rlp7xk4fp84vw4xx-borgbackup-1.1.10
ls -la /nix/store/4m7p421h0831sq45rlp7xk4fp84vw4xx-borgbackup-1.1.10/bin
```


</details>
<br/>


cc @flokli @dotlambda for testing.

r-ryantm committed:
borgbackup: 1.1.9 -> 1.1.10

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/borgbackup/versions

c0bw3b commented:
@GrahamcOfBorg test borgbackup

c0bw3b commented:
Announce: https://github.com/borgbackup/borg/releases/tag/1.1.10
Changelog: https://github.com/borgbackup/borg/blob/1.1.10/docs/changes.rst#version-1110-2019-05-16

> bundle latest supported msgpack-python release (0.5.6), remove msgpack-python from setup.py install_requires 

So the python3 override may not be needed anymore.

flokli committed:
borgbackup: remove custom msgpack-python override, use bundled version

https://github.com/borgbackup/borg/blob/1.1.10/docs/changes.rst#version-1110-2019-05-16
mentions borgbackup now shipping a supported msgpack-python release, and
strongly encourages using that one.

So don't pass any msgpack-python into the build, and use the provided
one.

flokli commented:
I updated the derivation to not pass in any msgpack-python, instead use the one bundled with borgbackup itself, as the former seems to be strongly discouraged:

> optionally, we still support using an external msgpack (see hints in setup.py), but this requires solid requirements management within distributions and is not recommended. borgbackup will break if you upgrade msgpack to an unsupported version.

with that change, borgbackup vm tests still pass, so this seems to work :-)
